content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -16,17 +16,276 @@ further_reading:
 
-{{< partial name="apm/apm-context-propagation" >}}
+## Configuration
 
+If you need to customize the trace context propagation configuration, there are several environment variables you can use to configure the formats that are used for reading and writing distributed tracing headers. To enable a specific propagator, make sure to use the corresponding configuration value for the tracing library, as outlined in the **Language support** section.
 
-<br>
+<div class="alert alert-info">
+If multiple propagators are enabled, the extraction attempt is done in the specified order, and the first valid trace context is used to continue the distributed trace. If additional valid trace contexts are found, the tracing information will be recorded as individual span links.</div>
+
+### Recommended Datadog configuration
+
+`DD_TRACE_PROPAGATION_STYLE`
+: Specifies propagators (in a comma-separated list) to be used for trace context propagation. This may be overridden by the extract-specific or inject-specific configurations. <br>
+**Default:** `datadog,tracecontext` for language SDK's, `tracecontext,datadog` for Envoy and NGINX procies
+
+### Recommended OpenTelemetry configuration
+
+`OTEL_PROPAGATORS`
+: Specifies propagators (in a comma-separated list) to be used for trace context propagation. This configuration takes the lowest precedence and will be ignored if any other Datadog propagation environment variable is set.
+
+### Advanced configuration
+
+In the majority of scenarios, you will want to both send and receive trace context headers using the same format. However, if your service needs to accept trace context headers in one format and send them in another format, you can customize them with the following configurations.
+
+`DD_TRACE_PROPAGATION_STYLE_EXTRACT`
+: Specifies propagators (in a comma-separated list) to be used only for trace context extraction. This configuration takes the highest precedence over all other configurations for configuring the extraction propagators.
+
+`DD_TRACE_PROPAGATION_STYLE_INJECT`
+: Specifies propagators (in a comma-separated list) to be used only for trace context injection. This configuration takes the highest precedence over all other configurations for configuring the injection propagators.
+
+## Language support
+
+{{< tabs >}}
+
+{{% tab "Java" %}}
+
+### Supported propagators
+The Datadog Java SDK maintains several propagators for passing trace context information in different formats:
+
+| Propagator             | Configuration Value |
+|------------------------|---------------------|
+| Datadog                | `datadog`           |
+| [W3C Trace Context][2] | `tracecontext`      |
+| [B3 Single][3]         | `b3 single header`  |
+|                        | `b3single`          |
+| [B3 Multi][4]          | `b3multi`           |
+|                        | `b3` (deprecated)   |
+| None                   | `none`              |
```

Comment:
Now may be a good time to standardize on the naming here across languages. We can have our table on the main page be the intersection of all languages and then language-specific tabs can describe "deprecated" behavior

To be clear, we can standardize around the following values across all languages
`datadog, tracecontext, b3single, b3multi, none` and mark the other aliased names as deprecated

Reply:
I think this is a good idea and we could standardize on `datadog, tracecontext, b3multi, none` but don't have a standardized name for the `b3single` style. We could enter this into the table and say that it's language-dependent?

@brett0000FF I wonder what you think?

Reply:
I've added an experimental commit to add a consolidated table, in addition to the language-specific tables: 9c42a0d3643c8a130d184bb7005763c02d98b4a6